test/redeem-script-parser-test.js

```diff
@@ -48,19 +49,56 @@ const validateStandardRedeemScriptFormat = (redeemScript, pubKeys) => {
     checkPubKeysIncludedInRedeemScript(pubKeys, redeemScript);
 }
 
-const validateP2shErpRedeemScriptFormat = (p2shErpRedeemScript, pubKeys, erpPubKeys) => {
-    // First byte is OP_NOTIF
-    expect(p2shErpRedeemScript.substring(0,2)).to.be.eq(decimalToHexString(opcodes.OP_NOTIF));
-    // Second to last byte is OP_CHECKMULTISIG
-    expect(p2shErpRedeemScript.slice(-4).substring(0,2)).to.be.eq(decimalToHexString(opcodes.OP_CHECKMULTISIG));
-    // Last byte is OP_ENDIF
-    expect(p2shErpRedeemScript.slice(-2)).to.be.eq(decimalToHexString(opcodes.OP_ENDIF));
-    
-    // Public keys should be in the redeem script
-    checkPubKeysIncludedInRedeemScript(pubKeys, p2shErpRedeemScript);
-    checkPubKeysIncludedInRedeemScript(erpPubKeys, p2shErpRedeemScript);
+const validateP2shErpRedeemScriptFormat = (p2shErpRedeemScript, pubKeys, erpPubKeys, csvValue) => {
+
+    const OP_M = parseInt(pubKeys.length / 2) + 1;
```

Comment:
```suggestion
    const OP_M = decimalToOpCode(parseInt(pubKeys.length / 2) + 1);
```
Wouldn't that simplify it?

Reply:
Resolved